setupKAppNavTestEnv.sh

```diff
@@ -0,0 +1,172 @@
+#!/bin/bash
+####################################################
+#
+# setupKAppNavTestEnv.sh for Kubernetes Application Navigator
+#
+####################################################
+
+# check if user wants help
+if [ "$#" -lt 5 ] || [  x$1 = 'x' ] || [  x$1 = x'?' ] || [  x$1 = x'--?' ] || [  x$1 = x'-?' ] || [  x$1 = x'--help'  ] || [  x$1 = x'help' ]; then
+    echo "================================================================"
```

Comment:
Juni, please remove this comment.

Reply:
What message I should remove?

Reply:
The one that starts with "Note:".  Lines 10-12.

Reply:
ok